GVFS/GVFS.FunctionalTests/Categories.cs

```diff
@@ -6,6 +6,8 @@ public static class Categories
         public const string FastFetch = "FastFetch";
         public const string GitCommands = "GitCommands";
 
+        public const string RepositoryMountsSameFileSystem = "RepositoryMountsSameFileSystem";

```

Comment:
Should there be a change in `Program.cs` to start including/excluding the `RepositoryMountsSameFileSystem` category based on what platform you're on?

Reply:
The applicable platform is Linux, and since that's not in master yet, the other two platforms both can use the default.

Reply:
Ahh okay that's right, thanks!